### ISSUE TITLE
feat(api): add support for web sockets or web hooks for blobs, txs and blocks

### ISSUE THREAD

**Issue by PJColombo** [Blobscan/blobscan#335]
It would be interesting to have support for web hooks or web sockets for new blobs, blob transactions or blocks and maybe even parametrising them with a specific rollup or rollups

**Comment by tunnckoCore** [comment]
Yeah, maybe something like Svix  or Hooks0 can be used. Or as simple as Upstash Qstash to signal to whoever is registered for a webhook through some basic form or process.
